doc/user/content/transform-data/source-troubleshooting.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Troubleshooting Sources and Sinks"
+description: ""
+menu:
+  main:
+    name: "Troubleshooting sources and sinks"
+    identifier: source-troubleshooting-transform
+    parent: transform
```

Comment:
Ah I wonder if the slow/unresponsive queries should go under a troubleshooting doc in serve-results then, rather than transform-data. And the file here "dataflow-troubleshooting.md" stays in transform-data.

Reply:
That's...a great question. The fact that I don't like the current navigation (🫠 ) makes it harder to answer, but it makes sense to place it under `Serve results` for now, yeah!